wandb/deploys/launch-agent.yaml

```diff
@@ -0,0 +1,86 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: wandb-launch-agent
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: launch-agent-demo
+  namespace: wandb-launch-agent
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: launch-agent-demo
+  template:
+    metadata:
+      labels:
+        app: launch-agent-demo
+    spec:
+      securityContext:
+        fsGroup: 107    # Group ID of docker group on k8s nodes. Also 412
+      containers:
+      - name: launch-agent-demo
+        image: launch-agent-container:v0
+        resources:
+          limits:
+            memory: "2Gi"
+            cpu: "1000m"
+        env:
+        - name: WANDB_PROJECT
+          valueFrom:
+            configMapKeyRef:
+              name: wandb-setup
+              key: wandb-project
+        - name: WANDB_BASE_URL
+          valueFrom:
+            configMapKeyRef:
+              name: wandb-setup
+              key: wandb-base-url
+        - name: WANDB_API_KEY
+          valueFrom:
+            secretKeyRef:
+              name: wandb-api-key
+              key: password
+        securityContext:
+          privileged: true
+          runAsNonRoot: false # added
+          runAsUser: 0
+          capabilities: 
+            drop:
+              - all
+          allowPrivilegeEscalation: true
```

Comment:
Why do we allow privilege escalation?  This would only be needed if we setup `sudo` in the container and wanted to be able to sudo...